pmlc/conversion/tile_to_pxa/tile_to_pxa.cc

```diff
@@ -417,12 +417,12 @@ static Value createInit(OpBuilder &builder, Location loc, Type type,
     }
     case AggregationKind::min: {
       auto value = convertFloatUsingType(
-          llvm::APFloat(std::numeric_limits<double>::infinity()), floatType);
+          llvm::APFloat::getInf(llvm::APFloat::IEEEdouble(), false), floatType);
```

Comment:
Ah, now try:
```
auto value = llvm::APFloat::getInf(type.getFloatSemantics(), false);
```

Reply:
Doesn't immediately work, but I'll poke around (I get `error: no member named 'getFloatSemantics' in 'mlir::Type'`)

Reply:
Sorry, I meant `floatType.getFloatSemantics()` (it's on the `FloatType`, but not on `Type`)

Reply:
That did it. Thanks for figuring that out!